pages/latest.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import ProjectList from '../components/project-list/project-list.jsx';
+import ProjectLoader from '../components/project-loader/project-loader.jsx';
 
 export default React.createClass({
```

Comment:
do we even need a full component here or can we just make this a stateless function?
```
export default function (props) {
  return <div><ProjectLoader/></div>;
};
```